plugin/file/zone.go

```diff
@@ -124,6 +124,20 @@ func (z *Zone) Insert(r dns.RR) error {
 // Delete deletes r from z.
 func (z *Zone) Delete(r dns.RR) { z.Tree.Delete(r) }
 
+// GetFile retrieves the file path in a safe way
+func (z *Zone) GetFile() string {
```

Comment:
Usually In Go you drop the Get, so this just becomes File.

other than that, lgtm